lib/helpers/px-to-rem.js

```diff
@@ -0,0 +1 @@
+module.exports = (value) => Math.round((value/16)*10000)/10000;
```

Comment:
Not sure this formula is correct. I found couple examples [here](https://codepen.io/levchenkod/pen/oYbWMB) and [here](https://gist.github.com/miladd3/17ed397bddd3d0333035c5a5d6c35a8a), which calculate it as `(1 / 16) * value`

Reply:
I actually don't think this is possible. Both of these methods are based on a base font size, because `rem` is based on a base font-size in itself. So there's not calculation possible without making the `16` configurable.

Taking a step back, in what scenario would a user ever want the `width` and `height` set in `rem`s? I understand the use-case for `px`, but don't see why we would want to support `rem`s as well.

Reply:
yeah, it seems like the font size would have to be passed down as a prop, but I agree changing svg sprite every time default font changes - questionable. imo, `px` is the way to go, but maybe I just didn't have the case where `rem` is a better use case.